### PR TITLE
Issue #16, 18 19

### PR DESCRIPTION
**Pull request by KevinZhiyuanDing**
Silkscreen text size was increased.
Potentiometer and encoder spacing were changed to be less cramped.
Change to rotary encoder input pin made consecutive preserved.
image

### CODE REVIEW

**Comment by JamesZhZhang** [comment]
Made a new commit to fix some trace routing for DRC. I had a couple notes I'd like to review in tomorrow's meeting too before we merge

**Comment by JamesZhZhang** [comment]
I think it's best to not merge this for the time being since we have a few new issues to tackle from Saturday. I think it'd be a good idea to work on all those issues on branches for the time being and merge at the end.

**Review by LaPommeCosmique — APPROVED** [review]

**Pull request merged**
